plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduMetadata.java

```diff
@@ -478,9 +485,25 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
     public Optional<ConstraintApplicationResult<ConnectorTableHandle>> applyFilter(ConnectorSession session, ConnectorTableHandle table, Constraint constraint)
     {
         KuduTableHandle handle = (KuduTableHandle) table;
+        TupleDomain<ColumnHandle> tupleDomain = constraint.getSummary();
```

Comment:
Does this change adds support for filter pushdown for Data type?

Reply:
Have removed this change.